packages/prisma/schema.prisma

```diff
@@ -402,6 +402,7 @@ model Team {
   platformOAuthClient   PlatformOAuthClient[]
   smsLockState          SMSLockState            @default(UNLOCKED)
   platformBilling       PlatformBilling?
+  auditLog              AuditLog[]
```

Comment:
Is the relationship needed at all? We can ditch the relationship and still query by teamId don't we?

Reply:
Yes. It is needed because We onDelete Cascade expects opposite relation from auditlog table. If we omit onDelete cascade then If a team gets deleted the audit logs of that team stays. So inorder to also delete the auditlogs belonging to the deleted team must be also deleted.

Reply:
I think we want to keep the logs even if the team has been deleted. That's the point of having an audit log doesn't it?

Reply:
Great. Done.